ocfweb/docs/docs/staff/backend/rt.md

```diff
@@ -0,0 +1,57 @@
+[[!meta title="Request Tracker"]]
+
+[**Request Tracker**](https://rt.ocf.berkeley.edu/) is the ticketing system
+used by the OCF. It is the main way of keeping track of OCF-related activity.
+Tickets are currently made either by staff to propose things to talk about
+or by emails from the Hostmaster or Help mailing lists.
+
+## Queues
+Tickets are assigned to queues, or organized boards. Manually-created
+tickets are found under:
+- *bod* for meeting topics
+- *bureaucracy* for officer-related issues
+- *operations* for Operations Strategist work (opstaff)
+- *projects* for long-term activities
+- *techtalks* for Tech Talk ideas and organization
+- *todo* for shorter-term items
+
+Other tickets are created automatically from emails:
+- *devnull* for devnull@ emails
+- *help* for questions from users (help@)
+- *hostmaster* for requests for virtual hosting ([form](https://www.ocf.berkeley.edu/account/vhost/))
+- *security* for vulnerability detections from ISP
```

Comment:
i don't think this distinction is really accurate, i seldom use the web UI and instead just send emails

as-written this might confuse people into thinking they're set up specially somehow

probably better to just say the thing in my comment above